src/metabase/api/database.clj

```diff
@@ -576,15 +584,25 @@
   "Returns a list of all the schemas found for the database `id`"
   [id]
   (api/read-check Database id)
-  (->> (db/select-field :schema Table :db_id id)
+  (->> (db/select-field :schema Table :db_id id {:order-by [[:%lower.schema :asc]]})
```

Comment:
Would adding `:active true` here also fix #9781

Reply:
good catch @flamber, I'll make that change 👍

Reply:
All credit to @lindsay-stevens - I was just reading your code and made 1+1 :wink:

Reply:
thank you both!